plugins/@grouparoo/mysql/src/lib/export/exportRecord.ts

```diff
@@ -1,13 +1,22 @@
 import { ExportRecordPluginMethod } from "@grouparoo/core";
 import { MySQLConnection } from "../connect";
+import { checkOptionsIntegrity } from "./destinationMappingOptions";
 
 export const exportRecord: ExportRecordPluginMethod<MySQLConnection> = async ({
   connection,
   destination,
+  destinationOptions,
   export: { newRecordProperties, oldRecordProperties, newGroups, toDelete },
 }) => {
+  checkOptionsIntegrity(destinationOptions);
+  if (
+    !destinationOptions.groupsTable ||
+    !destinationOptions.groupForeignKey ||
+    !destinationOptions.groupColumnName
+  ) {
+    newGroups = [];
+  }
```

Comment:
I guess that works!
This will have the side-effect of deleting any previously-set groups you had saved in the table